server/supernova/sc/sc_osc_handler.cpp

```diff
@@ -728,7 +728,8 @@ void sc_osc_handler::tcp_connection::handle_message() {
 
 
 void sc_osc_handler::start_tcp_accept(void) {
-    tcp_connection::pointer new_connection = tcp_connection::create(tcp_acceptor_.get_io_service());
+    tcp_connection::pointer new_connection =
+        tcp_connection::create((boost::asio::io_context&)tcp_acceptor_.get_executor().context());
```

Comment:
why cast here? according to the documentation `context()` already returns an `io_context&` (https://www.boost.org/doc/libs/1_70_0/doc/html/boost_asio/reference/io_context__executor_type/context.html)